auth_test.go

```diff
@@ -102,6 +102,20 @@ func postAuthSuccess(_ context.Context, _ *snowflakeRestful, _ *http.Client, _ *
 	}, nil
 }
 
+func postAuthSuccessWithServerVersion(_ context.Context, _ *snowflakeRestful, _ *http.Client, _ *url.Values, _ map[string]string, _ bodyCreatorType, _ time.Duration) (*authResponse, error) {
```

Comment:
Same here. Use a diff file (unles we plan to upstream this)

Reply:
I do plan to send this upstream eventually, but I can move it for now